playbook/spec/pb_kits/playbook/kits/timestamp_spec.rb

```diff
@@ -128,7 +128,7 @@
         it "returns time elaspsed string including user's name" do
           timestamp = DateTime.new(2020, 10, 10, 20, 30, 0o0).in_time_zone("America/New_York").freeze
 
-          expect(subject.new(timestamp: timestamp, variant: variant, show_user: show_user, text: name).send(:format_elapsed_string)).to eq("Last updated by #{name} #{time_ago_in_words(timestamp)} ago")
+          expect(subject.new(timestamp: timestamp, variant: variant, show_user: show_user, text: name).send(:format_elapsed_string)).to eq("Last updated by #{name} #{subject.new.send(:pb_time_ago, timestamp)} ago")
```

Comment:
Why `subject.new.send` instead of the function?

Reply:
Adding note for clarity. Due to how private functions work in rails, we can't just directly call them. They need to be called via the object/model they are located in.